images/extension-builders/tinygo/commands.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Tetrate
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+extension_build()  {
+	tinygo build -o "$1" -scheduler=none -target wasi main.go
+}
+
+extension_test()  {
+	go test -tags=proxytest -v ./...
+}
+
+extension_clean()  {
```

Comment:
After I run `getenvoy extension clean`, here is what I still have on my machine:

```shell
tree -a

.
├── .getenvoy
│   └── extension
│       ├── extension.yaml
│       └── toolchains
│           └── default.yaml
├── build
├── extension.wasm
├── go.mod
├── go.sum
├── main.go
└── main_test.go

4 directories, 7 files
```

Can we remove both `build/` dir and `extension.wasm` file as part of `getenvoy extension clean` ?

BTW, why not move `extension.wasm` into `build/` ?

Reply:
fixed in https://github.com/tetratelabs/getenvoy/pull/104/commits/45a02f3d8be384ae359161c39478e4d2c7f6de15 and https://github.com/tetratelabs/getenvoy/pull/104/commits/b9024b03272fdc0154bc100bbf84b6978afb8b1c